js/classes/game.js

```diff
@@ -1,48 +1,50 @@
 class Game {
- constructor(round, players, gameState) {
+  constructor(round, players, gameState) {
    this.round = 1;
    this.gameState = 'standby';
    this.players = players;
- }
+  }
 
+  init() {
+    domUpdates.updateHostPrompt('Player One, enter your name below!')
+    // playerOne = new Player(undefined, undefined, undefined, 1);
+    // playerTwo = new Player(undefined, undefined, undefined, 2);
+    // playerThree = new Player(undefined, undefined, undefined, 3);
+    // domUpdates.populateGameBoard();
+  }
 
- // appendNamesAndScores(playerOne, playerTwo, playerThree) {
- //     document.querySelector('.player-one-name').innerHTML = playerOne.playerName;
- //     document.querySelector('.player-two-name').innerHTML = playerTwo.playerName;
- //     document.querySelector('.player-three-name').innerHTML = playerThree.playerName;
- //     document.querySelector('.player-one-score').innerHTML = playerOne.score;
- //     document.querySelector('.player-two-score').innerHTML = playerTwo.score;
- //     document.querySelector('.player-three-score').innerHTML = playerThree.score;
- // };
+  incrementTurn() {
+    if (playerIndex === 2) {
+      playerIndex = 0;
+    } else {
+      playerIndex++;
+    }
```

Comment:
Curious how you're using this method. `playerIndex` is not currently an instance property on your game. Is it a global variable? If so, it probably shouldn't be. If not, I'm not sure where you would be gaining access to `playerIndex` -- youre not passing it in as a parameter, and you're not returning it from this method.